# FAIL

fn f() -> int {
  let x = 10
}
